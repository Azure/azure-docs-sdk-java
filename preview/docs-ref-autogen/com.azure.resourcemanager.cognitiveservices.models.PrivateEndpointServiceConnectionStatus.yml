### YamlMime:JavaType
uid: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "Defines values for PrivateEndpointServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.cognitiveservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cognitiveservices:1.0.0-beta.1
