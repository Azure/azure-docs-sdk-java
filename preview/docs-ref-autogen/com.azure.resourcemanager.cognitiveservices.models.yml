### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.cognitiveservices.models"
fullName: "com.azure.resourcemanager.cognitiveservices.models"
name: "com.azure.resourcemanager.cognitiveservices.models"
summary: "Package containing the data models for CognitiveServicesManagementClient. Cognitive Services Management Client."
classes:
- "com.azure.resourcemanager.cognitiveservices.models.CheckDomainAvailabilityParameter"
- "com.azure.resourcemanager.cognitiveservices.models.CheckSkuAvailabilityParameter"
- "com.azure.resourcemanager.cognitiveservices.models.CheckSkuAvailabilityResult"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountApiProperties"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountListResult"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountProperties"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountSkuChangeInfo"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesResourceAndSku"
- "com.azure.resourcemanager.cognitiveservices.models.Encryption"
- "com.azure.resourcemanager.cognitiveservices.models.Identity"
- "com.azure.resourcemanager.cognitiveservices.models.IpRule"
- "com.azure.resourcemanager.cognitiveservices.models.KeySource"
- "com.azure.resourcemanager.cognitiveservices.models.KeyVaultProperties"
- "com.azure.resourcemanager.cognitiveservices.models.MetricName"
- "com.azure.resourcemanager.cognitiveservices.models.NetworkRuleAction"
- "com.azure.resourcemanager.cognitiveservices.models.NetworkRuleSet"
- "com.azure.resourcemanager.cognitiveservices.models.OperationDisplayInfo"
- "com.azure.resourcemanager.cognitiveservices.models.OperationEntityListResult"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpoint"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnectionProperties"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateLinkResource"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateLinkResourceProperties"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.cognitiveservices.models.ProvisioningState"
- "com.azure.resourcemanager.cognitiveservices.models.PublicNetworkAccess"
- "com.azure.resourcemanager.cognitiveservices.models.QuotaUsageStatus"
- "com.azure.resourcemanager.cognitiveservices.models.RegenerateKeyParameters"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkuRestrictionInfo"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkuRestrictions"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkuRestrictionsReasonCode"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkusResult"
- "com.azure.resourcemanager.cognitiveservices.models.Sku"
- "com.azure.resourcemanager.cognitiveservices.models.SkuCapability"
- "com.azure.resourcemanager.cognitiveservices.models.SkuTier"
- "com.azure.resourcemanager.cognitiveservices.models.UnitType"
- "com.azure.resourcemanager.cognitiveservices.models.Usage"
- "com.azure.resourcemanager.cognitiveservices.models.UserAssignedIdentity"
- "com.azure.resourcemanager.cognitiveservices.models.UserOwnedStorage"
- "com.azure.resourcemanager.cognitiveservices.models.VirtualNetworkRule"
enums:
- "com.azure.resourcemanager.cognitiveservices.models.IdentityType"
- "com.azure.resourcemanager.cognitiveservices.models.KeyName"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkuRestrictionsType"
interfaces:
- "com.azure.resourcemanager.cognitiveservices.models.Accounts"
- "com.azure.resourcemanager.cognitiveservices.models.CheckDomainAvailabilityResult"
- "com.azure.resourcemanager.cognitiveservices.models.CheckSkuAvailabilityResultList"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.Definition"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.Blank"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithKind"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithSku"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.DefinitionStages.WithTags"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.Update"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages.WithKind"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages.WithProperties"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages.WithSku"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccount.UpdateStages.WithTags"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountEnumerateSkusResult"
- "com.azure.resourcemanager.cognitiveservices.models.CognitiveServicesAccountKeys"
- "com.azure.resourcemanager.cognitiveservices.models.OperationEntity"
- "com.azure.resourcemanager.cognitiveservices.models.Operations"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnection.UpdateStages.WithProperties"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.cognitiveservices.models.PrivateLinkResources"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceProviders"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSku"
- "com.azure.resourcemanager.cognitiveservices.models.ResourceSkus"
- "com.azure.resourcemanager.cognitiveservices.models.UsagesResult"
metadata: {}
package: "com.azure.resourcemanager.cognitiveservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cognitiveservices:1.0.0-beta.1
