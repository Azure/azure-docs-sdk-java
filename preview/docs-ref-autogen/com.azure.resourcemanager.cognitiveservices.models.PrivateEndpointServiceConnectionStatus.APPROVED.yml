### YamlMime:JavaMember
uid: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.cognitiveservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.cognitiveservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-cognitiveservices:1.0.0-beta.1
